CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Changelog
 * update all tests using the filter parameter instead of deprecated types, keys, values ([#98])
 * update some default parameter values in swagger UI to slightly more sensible examples ([#113])
 * restructure packages and classes within the controller and output packages ([#117])
+* limit the length of coordinates in response OSM features to 7 ([#138])
```

Comment:
hm, that's not quite correct. we are rounding to 7 decimal places, not _limiting the length to 7_. e.g. `49.4080159` has a "length" of 9 digits (or 10 characters when counting the `.`).

```suggestion
* round coordinates of returned OSM features to 7 decimal places ([#138])
```

Reply:
I was thinking about "precision" but apparently wrote "length" for some reason 🙄 
I'll take your suggestion yeah.